### PR TITLE
Adds "Summon Magic," summon guns alternative for wizard

### PR DESCRIPTION
**Pull request by JustSumBody**
Saw this when I was looking at the other summon guns PR and thought it might be neat to bring in(back? I couldn't find any evidence we removed it but it looked like we had it at one point)
Basically it's summon guns but with one use spellbooks/artifacts in place of guns, all the spellbooks are the ones that don't need robes. I might be missing one or two. Should some of the ones that need robes be thrown in?
As for artifacts, I'm not sure if I should just throw them all in or what. Right now there's just the scrying orb, wizard hardsuit, and staff of change.
I don't care much about any of the specifics, but I did like the idea when I read it.

### CODE REVIEW

File: code/game/gamemodes/wizard/rightandwrong.dm [hunk 1]
```diff
@@ -22,7 +22,7 @@
 				to_chat(H, "<B>Objective #[obj_count]</B>: [OBJ.explanation_text]")
 				obj_count++
 		var/randomizeguns = pick("taser","egun","laser","revolver","detective","smg","nuclear","deagle","gyrojet","pulse","silenced","cannon","doublebarrel","shotgun","combatshotgun","mateba","smg","uzi","crossbow","saw","hecate","osipr","gatling","bison","ricochet","spur","nagant","beegun")
-		var/randomizemagic = pick("fireball","smoke","blind","mindswap","forcewall","knock","horsemask","charge","wandnothing", "wanddeath", "wandresurrection", "wandpolymorph", "wandteleport", "wanddoor", "wandfireball", "staffchange", "staffhealing", "armor", "scrying")
+		var/randomizemagic = pick("fireball","smoke","blind","mindswap","forcewall","knock","horsemask","blink","subjugate","charge","staffchange","armor","scrying")
```
Comment (kol1th) [inline code/game/gamemodes/wizard/rightandwrong.dm]: Did you mean to leave charge in here?

**Comment by clusterfack** [comment]
Summon magic is summon guns meme taken too far

File: code/game/gamemodes/wizard/artifacts.dm [hunk 2]
```diff
@@ -108,6 +108,22 @@
...
+	abbreviation = "SM"
+
+/datum/spellbook_artifact/summon_magic/can_buy()
+	//Can't summon magic during ragin' mages
+	return !ticker.mode.rage
+
+/datum/spellbook_artifact/summon_magic/purchased(mob/living/carbon/human/H)
+	..()
+
+	H.rightandwrong(1)
```
Comment (gbasood) [inline code/game/gamemodes/wizard/artifacts.dm:125]: N O

**Review by gbasood — CHANGES_REQUESTED** [review]
This is literally just a purchased spellbook to give everyone summon guns
I liked this idea at first because I thought it'd give random crew members spellbooks
But then I saw the code
I have never been more disappointed

File: code/game/gamemodes/wizard/artifacts.dm (hunk 2 above)
Comment (gbasood) [reply code/game/gamemodes/wizard/artifacts.dm:125]: oh wait im wrong

**Review by gbasood — APPROVED** [review]

**Comment by Intigracy** [comment]
We had it at some point and it got commented out for whatever reason.

Spellbooks > guns

**Pull request merged**